week-3/todo-list/script.js

```diff
@@ -1,25 +1,96 @@
+// function populateTodoList(todos) {
+//   let list = document.getElementById("todo-list");
+//   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+// }
+
+// // These are the same todos that currently display in the HTML
+// // You will want to remove the ones in the current HTML after you have created them using JavaScript
+// let todos = [
+//   { task: "Wash the dishes", completed: false },
+//   { task: "Do the shopping", completed: false },
+// ];
+
+// populateTodoList(todos);
+
+// // This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
+// const button = document.querySelector("#add_todo");
+// button.addEventListener("click", addNewTodo);
+// const ul = document.getElementById("unorder_list");
+// // console.log(ul);
+// function addNewTodo(event) {
+//   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
+//   event.preventDefault();
+//   // Write your code here... and remember to reset the input field to be blank after
+//   const input = document.querySelector("#input").value;
+//   const nweToDo = { task: input };
+//   // const li = document.createElement("li");
+//   // ul.appendChild(li);
+
+//   // li.textContent = input;
+// }
+
+// // Advanced challenge: Write a fucntion that checks the todos in the todo list and deletes the completed ones (we can check which ones are completed by seeing if they have the line-through styling applied or not).
+// function deleteAllCompletedTodos() {
+//   // Write your code here...
+// }
+
+
+
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
+  todos.forEach((todo) => {
```

Comment:
As per the comment above, you don't have a list called `todos` so this function won't do anything.